modules/cluster-resizing/pages/shrink-a-cluster.adoc

```diff
@@ -7,6 +7,9 @@ Shrinking a cluster can make sense for a few reasons:
 
 Shrinking a cluster removes nodes from the cluster. The data stored on those nodes will be redistributed to the remaining nodes.
 
+Shrinking an operational cluster is different from removing a failed node.
+If a node has failed in your cluster and needs to be removed, see xref:ha:remove-failed-node.adoc[].
```

Comment:
You may mention that user can do expansion after remove. Then they can replace the failed nodes.